_15thnight/models.py

```diff
@@ -1,10 +1,11 @@
 """Data Models."""
-import uuid
 from datetime import datetime, timedelta
+import uuid
```

Comment:
I agree with the grouping and that is what is used through the projects.

But within the subgroups I list `import ...` first before `from ... import ...`